### ISSUE TITLE
Sync:// transport no longer works after installing kcs

### ISSUE THREAD

**Issue by arendjantetteroo** [alekitto/messenger-extra#7]
It returns :
```

In RegisterDoctrineEventsPass.php line 22:
                                                                                 
  [Symfony\Component\Config\Definition\Exception\InvalidConfigurationException]  
  "sync://" is not a valid URL     
```

This sync:// mode we use in dev mode for easier developing. If i use `sync://null` it seems to work, however `sync://` is what is used by symfony in their documentation. 

@alekitto should we do a return early if $url === 'sync://' in the Pass code (CheckDependencyPass is also affected). And maybe others if they are there.

**Comment by arendjantetteroo** [comment]
See #8

**Comment by arendjantetteroo** [comment]
Fixed by #9
